Apps/Admin/Server/Services/DelegationService.cs

```diff
@@ -0,0 +1,119 @@
+    using HealthGateway.Common.Data.ErrorHandling;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.Common.Factories;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Common.Services;
+    using HealthGateway.Database.Delegates;
+    using HealthGateway.Database.Models;
+    using Microsoft.Extensions.Configuration;
+
+    /// <inheritdoc/>
+    public class DelegationService : IDelegationService
+    {
+        private const string DelegationConfigSection = "Delegation";
+        private const string MaxDependentAgeKey = "MaxDependentAge";
+        private readonly IPatientService patientService;
+        private readonly IResourceDelegateDelegate resourceDelegateDelegate;
+        private readonly IMapper autoMapper;
+        private readonly int maxDependentAge;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DelegationService"/> class.
+        /// </summary>
+        /// <param name="configuration">The injected configuration provider.</param>
+        /// <param name="patientService">The injected patient service.</param>
+        /// <param name="resourceDelegateDelegate">The injected resource delegate delegate.</param>
+        /// <param name="autoMapper">The injected automapper provider.</param>
+        public DelegationService(
+            IConfiguration configuration,
+            IPatientService patientService,
+            IResourceDelegateDelegate resourceDelegateDelegate,
+            IMapper autoMapper)
+        {
+            this.patientService = patientService;
+            this.resourceDelegateDelegate = resourceDelegateDelegate;
+            this.autoMapper = autoMapper;
+            this.maxDependentAge = configuration.GetSection(DelegationConfigSection).GetValue(MaxDependentAgeKey, 12);
+        }
+
+        /// <inheritdoc/>
+        public async Task<DelegationResponse> GetDelegationInformationAsync(string phn)
+        {
+            // Get dependent patient information
+            RequestResult<PatientModel> dependentPatientResult = await this.patientService.GetPatient(phn, PatientIdentifierType.Phn).ConfigureAwait(true);
+
+            ValidationResult? validationResults = await new DelegationValidator(this.maxDependentAge).ValidateAsync(dependentPatientResult.ResourcePayload).ConfigureAwait(true);
+            if (!validationResults.IsValid)
+            {
+                throw new ProblemDetailsException(ExceptionUtility.CreateProblemDetails(MaxDependentAgeKey, HttpStatusCode.BadRequest, nameof(DelegationService)));
+            }
+
+            DelegationResponse delegationResponse = new();
+            if (dependentPatientResult.ResourcePayload != null)
+            {
+                PatientModel dependentPatientInfo = dependentPatientResult.ResourcePayload;
+                DependentInfo dependentInfo = this.autoMapper.Map<DependentInfo>(dependentPatientInfo);
+                dependentInfo.Protected = null;
```

Comment:
Default value doesn't need to be set (and the default will be false with the nullability change).